### ISSUE TITLE
Polyfill not working in Chrome

### ISSUE THREAD

**Issue by dkerzig** [filamentgroup/loadCSS#160]
Hey guys, I am using Chrome 49 (49.0.2623.110) and neither on my site nor on your preload sample-page the styles are loaded at all :/ I've tested it in Safari and everything works fine, and I am also 100% sure my site as worked in earlier Chrome versions (maybe 48) as well as your demo page. What could be the problem, any ideas?

**Comment by vijayaraghavanramanan** [comment]
One reason I can think of is that Chrome 49 only supports it with Experimental Web Platform features turned on but that it returns a true on feature detection. So it probably does nothing. I however don't see it happening in my tests.

**Comment by frxnz** [comment]
I can confirm this is an issue in Chrome 49. [cssrelpreload.js:10](https://github.com/filamentgroup/loadCSS/blob/master/src/cssrelpreload.js#L10) returns a false positive.

**Comment by frxnz** [comment]
Just noticed this is a dupe of #154. Resetting experimental flags solved the problem.

**Comment by scottjehl** [comment]
Yes, it's that flag. Not much we can do but it'll fix itself and won't show up for non-developers, in the meantime. Thanks!
